### ISSUE TITLE
What is CStr?

### ISSUE THREAD

**Issue by van-de-bugger** [Raku/doc#3827]
[`sub explicitly-manage`](https://docs.raku.org/routine/explicitly-manage):

    sub explicitly-manage($str) is export(:DEFAULT)

> This returns a `CStr` object for the given `Str`. If the string returned is passed to a NativeCall subroutine, it will not be freed by the runtime's garbage collector.

What is `CStr`? It appears only once in Raku documentation, on this page.

**Comment by JJ** [comment]
Right, good one. OTTOMH, I would say it's some implementation-specific class. But we definitely need to find out.

**Comment by jonathanstowe** [comment]
The `CStr` class is  internal to the `explicitly-manage` and is basically a wrapper for the underlying string with the `CStr` repr.  Really it appears that it being returned by the routine is a side-effect and there is not much that can be usefully done with it (it only has one method `encoding` which reflects the encoding of the original string.)
